packages/react-components/react-nav-preview/src/components/NavDrawer/useNavDrawer.ts

```diff
@@ -29,21 +28,20 @@ export const useNavDrawer_unstable = (props: NavDrawerProps, ref: React.Ref<HTML
     ref,
   );
 
-  const elementType = type === 'inline' ? InlineDrawer : OverlayDrawer;
-
   return {
     ...navState,
     components: {
-      root: elementType,
+      root: Drawer,
     },
-    root: slot.always<NavDrawerProps>(
+
+    root: slot.always<DrawerProps>(
       slot.resolveShorthand({
         ref,
         ...props,
         ...focusAttributes,
       }),
```

Comment:
won't this work without the `resolveShorthand`? after all, you're not passing any shorthand here in this case, you're spreading a well defined object,
```suggestion
      {
        ref,
        ...props,
        ...focusAttributes,
      },
```